ElectronicObserver/Window/Wpf/ExpeditionCheck/ExpeditionCheckRow.cs

```diff
@@ -34,7 +34,9 @@ public class ExpeditionCheckRow
 	public SolidColorBrush Fleet2BackgroundColor => GetBackgroundColor(Fleet2Result).ToBrush();
 	public SolidColorBrush Fleet3BackgroundColor => GetBackgroundColor(Fleet3Result).ToBrush();
 	public SolidColorBrush Fleet4BackgroundColor => GetBackgroundColor(Fleet4Result).ToBrush();
-
+	public SolidColorBrush ExpeditionNameBackgroundColor => GetBackground().ToBrush();
+	public SolidColorBrush ExpeditionIDBackgroundColor => GetBackground().ToBrush();
+	public SolidColorBrush ConditionBackgroundColor => GetBackground().ToBrush();
```

Comment:
Any reason why you split these backgrounds? None of these has any special handling, so you could probably just have a single `Background` property.